### PR TITLE
Add a tutorial on automated tests.

### PR DESCRIPTION
**Pull request by will-cromar**

### CODE REVIEW

File: doc/tutorials/test-templates-automated.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+This `CronJob` will automatically run and its metrics will be periodically collected by the Metrics Handler. Note that, by default, all TensorBoard summaries are collected by the Metrics Handler, but only cerain metrics have assertions.
+
+## Configuring metric aggregation
+
+You can configure metric aggregation and regression alerting with the `metricCollectionConfig` and `regressionTestConfig` field of `BaseTest`, respectively. For this example, we can set a target final accuracy of 98% for the test:
+
+```
+local mnist = base.BaseTest {
+  ...
+  metricCollectionConfig+: {
```
Comment (zcain117) [inline doc/tutorials/test-templates-automated.md:105]: add `"default_aggregation_strategies": ["final"],`

File: doc/tutorials/test-templates-automated.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+    metric_to_aggregation_strategy: {
+      epoch_sparse_categorical_accuracy: ['final'],
+    }
+  }
+  regressionTestConfig: {
```
Comment (zcain117) [inline doc/tutorials/test-templates-automated.md:110]: see https://github.com/GoogleCloudPlatform/ml-testing-accelerators/tree/master/metrics_handler#regression_test_config

This should be more like:

```
regression_test_config: {
  metric_success_conditions: {
    epoch_sparse_categorical_accuracy_final: {
      comparison: "greater",
      success_threshold: {
        fixed_value: 98.0
      }
    },
```

File: doc/tutorials/test-templates-automated.md [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+jsonnet -J ml-testing-accelerators/ mnist-cpu.jsonnet | kubectl create -f -
+```
+
+You can find deployed `CronJob` running on your [GKE workloads](https://console.cloud.google.com/kubernetes/workload) page. You can either wait for the `CronJob` trigger, or manually trigger a run of the cronjob with the following command:
+
+```
+kubectl create job --from=cronjob/tf-mnist-example-cpu tf-mnist-example-cpu-$(date +"%Y%m%d%H%M")
+```
+
+If you look at the beginning of the logs for your job, you'll see a message like the following from the `publsher` container:
```
Comment (zcain117) [inline doc/tutorials/test-templates-automated.md:133]: typo in "publsher"

File: doc/tutorials/test-templates-automated.md (hunk 2 above)
Comment (will-cromar) [reply doc/tutorials/test-templates-automated.md:115]: Done.

File: doc/tutorials/test-templates-automated.md (hunk 3 above)
Comment (will-cromar) [reply doc/tutorials/test-templates-automated.md]: Done.

File: doc/tutorials/test-templates-automated.md (hunk 1 above)
Comment (will-cromar) [reply doc/tutorials/test-templates-automated.md:110]: This is inherited from the base template

**Pull request merged**